src/util/evalUnits.ts

```diff
@@ -89,3 +89,8 @@ export function evalUnits(input: string, chainDecimals: number): [BN | null, str
   }
   return [numeric, EvalMessages.SUCCESS];
 }
+
+export function transformToBaseUnit(estFee: string, chainDecimals: number): string {
+  const est = parseInt(estFee) / 10 ** chainDecimals;
+  return est.toString();
```

Comment:
Any chance we can do it using `BN` library? Or manually? It's basically about padding with zeros and potentially adding `0.` in front. The reason I ask for this is because if `chainDecimals` are large we will again run into float issues and weird  values being displayed.

Example:
```
(235 / 10 ** 9).toString() == '2.35e-7'
```

Reply:
Im not sure that you will ever have 235 fee... In addition is there a chance that the chain will have way more than 9 decimals?

Reply:
It's not about our assumptions, but what Substrate is designed for. If we can make it always correct (and not only under some opinionated conditions) with a bit of more code, let's do it.

Ethereum has 18 decimals for instance and the fees are in `Gwei` so 20 Gwei should not be displayed as `(20 * 10**9/10**18).toString() === '2e-8'`